doc/odoc_for_authors.mld

```diff
@@ -663,6 +663,38 @@ v}
 In this case the reference text would be "Foo Bar" so the paragraph would read
 "See Foo Bar for details".
 
+Note that label names can contain [-] but if they do, you have to quote the label
+name in the reference link as follows:
+
+{v
+{2:foo-bar Foo Bar}
+...
+See {!label-"foo-bar"} for details
+v}
+
+You can use [_] instead of [-] to avoid this syntax.
+
+Sections can also be cross referenced from another page using the following syntax:
+{v
+You can find more information in the related section from that other page by
+following {{!page-that_other_page.that_section} this link}
+v}
+
+When cross referencing a section from a different page without a specified text
+for the link, the label will not be expanded into the corresponding section's
+title.
+For example, if you define the following section in {a.mld}:
+{v
+{1:some_label Some Grand Title}
+v}
+
+And then later reference it from {b.mld} as follows:
+{v
+See {!page-a.some_label}
+v}
+
+The paragraph will read "See some_label".
```

Comment:
We expect "See Some Grand Title" right ?

Reply:
If the behaviour was consistent with section links within the page yes, that's what we would expect.